doc/whats_new.rst

```diff
@@ -22,15 +22,24 @@ Current
 Changelog
 ~~~~~~~~~
 
+- Adding :class:`mne_connectivity.TemporalConnectivity`, :class:`mne_connectivity.SpectralConnectivity` and :class:`mne_connectivity.SpectroTemporalConnectivity` as a data structure to hold connectivity data, by `Adam Li`_ (:gh:`6`)
+
 Bug
 ~~~
 
 API
 ~~~
 
+- :func:`mne_connectivity.envelope_correlation` has a new required parameter, ``names``, which specifies the name of the signals passed in, by `Adam Li`_ (:gh:`6`)
```

Comment:
I would not require this. We can just have `names=None` behave like `map(range(n_signals), str)`

Reply:
Yes, especially for source points this is what you would  end up doing anyway